frontend/cypress/config/author-config.csv

```diff
@@ -0,0 +1,2 @@
+Repository's Location,Branch,Author's Git Host ID,Author's Emails,Author's Display Name,Author's Git Author Name,Ignore Glob List
+https://github.com/reposense/publish-RepoSense.git,master,yong24s,,Yong Hao,,
```

Comment:
Seems like this author config is not accurately recognizing Yong Hao's commits in `publish-RepoSense`, as it says 0 lines contributed when he has 148 lines. Perhaps look at this RepoSense documentation to specify multiple Git author names for one user.

Reply:
fixed!